utils/tfrecords/requirements.txt

```diff
@@ -0,0 +1,5 @@
+argparse>=1.1
+tensorflow
+pathlib
+Pillow
```

Comment:
Probably it is better to fix version of libraries to be sure that the script will work in the future.

Reply:
@nmanovic thank you for the note, I will fix them shortly.